azure/registry/variables.tf

```diff
@@ -0,0 +1,37 @@
+variable "project" {
+  description = "Project name"
+  type        = string
+}
+
+variable "environment" {
+  description = "Environment name"
+  type        = string
+}
+
+variable "resource_group_name" {
+  description = "Resource group name"
+  type        = string
+}
+
+variable "location" {
+  description = "Azure region"
+  type        = string
+}
+
+variable "sku" {
+  description = "ACR SKU"
+  type        = string
+  default     = "Standard"
+}
+
+variable "admin_enabled" {
+  description = "Enable admin user"
+  type        = bool
+  default     = true
+}
+
+variable "tags" {
+  description = "Resource tags"
+  type        = map(string)
+  default     = {}
```

Comment:
The default tags need to at least cover the project name and the environment (e.g., https://github.com/RADAR-base/RADAR-K8s-Infrastructure/blob/26c412a43c470e427fe417556478c91edca43ef4/cluster/variables.tf#L44).